storage/api/src/main/java/tech/pegasys/teku/storage/api/StorageUpdate.java

```diff
@@ -59,6 +61,9 @@ public StorageUpdate(
     this.stateRoots = stateRoots;
     this.optimisticTransitionBlockRootSet = optimisticTransitionBlockRootSet;
     this.optimisticTransitionBlockRoot = optimisticTransitionBlockRoot;
+    checkArgument(
```

Comment:
I saw it once in debugger but maybe I'm wrong. Can we have a case when we clear one set before with empty?